runtime/altair/src/lib.rs

```diff
@@ -1715,11 +1715,10 @@ impl pallet_order_book::Config for Runtime {
 }
 
 impl pallet_transfer_allowlist::Config for Runtime {
-	type CurrencyId = CurrencyId;
+	type CurrencyId = FilterCurrency;
```

Comment:
`Currency` is used in `pallet_transfer_allowlist` storages, so this would need migration. Is this pallet already used in centrifuge?

Reply:
Done in https://github.com/centrifuge/centrifuge-chain/pull/1693/commits/fc621f78c18b752c31f319e4584568ffed8a394c